reference/conan_v2_mode.rst

```diff
@@ -4,7 +4,7 @@
 CONAN_V2_MODE
 =============
 
-If defined in the environment, this variable will raise errors whenever a :ref:`Conan 2.0 <conan_v2>` deprecated feature
+If defined in the environment, this variable will raise errors whenever a :ref:`Conan 2.0 <conan2_migration_guide>` deprecated feature
```

Comment:
This env-var is not really supported anymore. The recommended way is to use the beta.
This can be dropped from the docs, IMO

Reply:
done in https://github.com/conan-io/docs/pull/2868/commits/986c699471c5560205a68ea26a0e2a9245616edf